documentation/frontend/guides/test.md

```diff
@@ -17,39 +17,45 @@ Storybook visual regression tests, read
 
 2. Run unit tests for the frontend.
 
-   ```console
```

Comment:
This is present in many other files too, would you mind opening an issue to fix these? It'll be a simple change that hopefully gives a new contributor a reason to set things up locally!

Reply:
I'm about to drop another PR that fixes these and adds a pre-commit check for it, actually!